main.go

```diff
@@ -61,7 +61,7 @@ func main() {
 				Name:        "nats-url",
 				Usage:       "NATS server URL (needs JetStream)",
 				Aliases:     []string{"n"},
-				Value:       "nats://localhost:4222",
+				Value:       "", // If empty URL, run without NATS
```

Comment:
We want Valtrack to be functional without NATS also. Assuming here if `nats-url` flag isn't set the user want to run without it.